src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -179,6 +179,16 @@ ExpressionPtr Visitor::processIriFunctionCall(
       return sparqlExpression::makeConvertToStringExpression(
           std::move(argList[0]));
     }
+    if (functionName == "dateTime") {
+      checkNumArgs(1);
+      return sparqlExpression::makeConvertToDateTimeExpression(
+          std::move(argList[0]));
+    }
+    if (functionName == "date") {
+      checkNumArgs(1);
+      return sparqlExpression::makeConvertToDateExpression(
+          std::move(argList[0]));
+    }
```

Comment:
In the `visit(BuiltinCallContext)` function, we have a lambda that allows this three liner to just be
`makeUnary(&makeConvertToDateExpression)` 

This would be a nice opportunity to also perform a similar cleanup for this rather lengthy function.